rdkit/Chem/Fingerprints/FingerprintMols.py

```diff
@@ -326,25 +319,21 @@ def _clusterInit(self):
     self.actName = ''
 
   def GetMetricName(self):
-    if self.metric == DataStructs.TanimotoSimilarity:
-      return 'Tanimoto'
-    elif self.metric == DataStructs.DiceSimilarity:
-      return 'Dice'
-    elif self.metric == DataStructs.CosineSimilarity:
-      return 'Cosine'
-    elif self.metric:
-      return self.metric
-    else:
-      return 'Unknown'
+    # DataStructs.TverskySimilarity: 'Tversky'
+    metricDict = {DataStructs.DiceSimilarity: 'Dice', 
```

Comment:
I will accept this change, but I don't think that this improves readability, isn't shorter, and it certainly is going to be slower (doesn't matter here since it's not on a critical path)